index.js

```diff
@@ -0,0 +1,6 @@
+sortRawData();
+buildComment();
+blog.truncateArticles();
+populateAuthorFilter();
+populateCategoriesFilter();
+$('#template').remove();
```

Comment:
From the technical requirements:
> Leave as little in the window (global) namespace as possible: attach functions to objects, etc.

Could these functions be part of the blog object?